pkg/mover/rewrite_rules.go

```
@@ -21,9 +22,16 @@ type RewriteRules struct {
 
 func (r *RewriteRules) Validate() error {
 	if r.Registry != "" {
-		_, err := name.NewRegistry(r.Registry, name.StrictValidation)
-		if err != nil {
-			return errors.New("registry rule is not valid")
+		if strings.Contains(r.Registry, "/") {
+			_, err := name.NewRepository(r.Registry, name.StrictValidation)
+			if err != nil {
+				return errors.New("registry rule is not valid")
```

```suggestion
				return fmt.Errorf("registry rule is not valid: %w", err)
```

---

I do not think we should hide the underlying error here